graphistry/feature_utils.py

```diff
@@ -147,13 +140,10 @@ def resolve_feature_engine(
 
     if feature_engine in ["none", "pandas", "dirty_cat", "torch"]:
         return feature_engine  # type: ignore
-
     if feature_engine == "auto":
-        has_dependancy_text_, _, _ = lazy_import_has_dependancy_text()
-        if has_dependancy_text_:
+        if deps.sentence_transformers:
```

Comment:
this seems unsafe... shouldn't each level check for all the dependencies needed for it?